### ISSUE TITLE
[Errno 113] Host is unreachable

### ISSUE THREAD

**Issue by ryryonline** [Tautulli/Tautulli#1157]
I have PlexPass enabled and each user needs to enter a pin to access the system. I believe this error is happening because PlexPy is trying to access Plex, but can't select a user.

Is there a way to work around this?

**Comment by JonnyWong16** [comment]
> I believe this error is happening because PlexPy is trying to access Plex, but can't select a user.

No, PlexPy works fine with Plex Home.

It is impossible to tell what your issue is without any of the relevant info. Please follow the proper guidelines. https://github.com/JonnyWong16/plexpy#issues

**Comment by ryryonline** [comment]
I definitely should've posted according the guidelines. I was looking for a quick answer to see if PlexPy worked with Plex Home. Once you confirmed that it did, I did some digging into my setup.

It turned out that my Docker system lets me install all of my dockers into different IPs rather than the same IP with different ports. However, when this is done, each docker closes itself off to the other dockers. 

In the PlexPy settings, instead of selecting the local IP of my Plex server, I put in the external URL to my Plex server and it worked. 

I appreciated the quick response which led to fixing my issue. I hope this eventually helps someone else out.
